src/main/java/org/sonar/plugins/clojure/sensors/kibit/KibitSensor.java

```diff
@@ -0,0 +1,59 @@
+package org.sonar.plugins.clojure.sensors.kibit;
+
+
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
+
+import org.sonar.api.batch.sensor.Sensor;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.SensorDescriptor;
+
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonar.plugins.clojure.language.ClojureLanguage;
+
+import org.sonar.plugins.clojure.sensors.AbstractSensor;
+import org.sonar.plugins.clojure.sensors.CommandRunner;
+import org.sonar.plugins.clojure.sensors.CommandStreamConsumer;
+import org.sonar.plugins.clojure.sensors.Issue;
+import org.sonar.plugins.clojure.settings.ClojureProperties;
+
+import java.util.List;
+
+public class KibitSensor extends AbstractSensor implements Sensor {
+
+    private static final Logger LOG = Loggers.get(KibitSensor.class);
+
+    private static final String KIBIT_COMMAND = "kibit";
+    private static final String[] LEIN_ARGUMENTS = {"kibit","-r", "markdown"};
```

Comment:
I was wondering if markdown here is really a good idea. The issues are generated with ```clojure, which makes it difficult to read on SonarQube. On the other hand, if we parse the issues using the default output:
```
At src/dk/salza/liq/window.clj:81:
Consider using:
  (dec (int (* (w0 :dk.salza.liq.window/rows) amount)))
instead of:
  (- (int (* (w0 :dk.salza.liq.window/rows) amount)) 1)

At src/dk/salza/liq/window.clj:302:
Consider using:
  (update-in
    p1__4661879#
    [:dk.salza.liq.window/rows]
    +
    (w0 :dk.salza.liq.window/rows)
    1)
instead of:
  (assoc
    p1__4661879#
    :dk.salza.liq.window/rows
    (+
      (p1__4661879# :dk.salza.liq.window/rows)
      (w0 :dk.salza.liq.window/rows)
      1))
```
It will look better.

Reply:
This is changed now to standard output instead of markdown formatted.